src/components/tagsInput/index.js

```diff
@@ -310,16 +351,21 @@ export default class TagsInput extends BaseComponent {
   }
 
   render() {
-    const {disableTagRemoval, containerStyle, hideUnderline} = this.getThemeProps();
+    const {disableTagRemoval, containerStyle, hideUnderline, onTagNotValidate} = this.getThemeProps();
     const tagRenderFn = disableTagRemoval ? this.renderTag : this.renderTagWrapper;
-    const {tags} = this.state;
+    const {tags, isTagNotValidate, tagIndexToRemove} = this.state;
 
     return (
       <View style={[!hideUnderline && styles.withUnderline, containerStyle]}>
         <View style={styles.tagsList}>
           {_.map(tags, tagRenderFn)}
           {this.renderTextInput()}
         </View>
+        <View>
```

Comment:
I think the error message should appear under the underline, in the design they don't show it. but better check with the UX to make sure.

Reply:
that 's how it shows in the zeplin, who exactly to check with ?

Reply:
talk to Or Gertman

Reply:
asked Or and she said its supposed to be above

Reply:
👍